Documentation/Wiki/Advanced-Features/Pip-Weight.md

```diff
@@ -0,0 +1,33 @@
+## Concept of weight

```

Comment:
>weight [](start = 14, length = 6)

Should we comment here that we have historic metadata and generally take that into account when running processes and expected load on the machine, but that this helps in case of absense?